<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>431</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="outerFrame">
      <property name="styleSheet">
       <string notr="true">color: rgb(123,123,169);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameCaption">
         <property name="styleSheet">
          <string notr="true">color: rgb(106, 110, 127);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QFrame" name="frameCaptionText">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labCaption">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Stryim</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QToolButton" name="btnMinimize">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>_</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnQuit">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string notr="true">X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="lay_appWindow">
         <property name="styleSheet">
          <string notr="true">color: rgb(39, 39, 88);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="margin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(123, 123, 146);</string>
            </property>
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1,2">
            <property name="margin">
             <number>13</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnCamPlay">
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCamStop">
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>stop</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <property name="spacing">
               <number>8</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioCamNone">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCamNone">
                <property name="text">
                 <string>none</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioCamError">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 0, 0);
</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCamError">
                <property name="text">
                 <string>error</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioCamIdle">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 255);
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCamIdle">
                <property name="text">
                 <string>idle</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioCamReady">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 0);
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCamReady">
                <property name="text">
                 <string>ready</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioCamAir">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 192, 0);
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelCamAir">
                <property name="text">
                 <string>air</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="Line" name="line">
              <property name="styleSheet">
               <string notr="true">color: rgb(230,230,230);</string>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>3</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(123, 123, 146);</string>
            </property>
            <property name="text">
             <string>Stream</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>13</number>
            </property>
            <property name="margin">
             <number>13</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>rtmp://</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editRtmpUrl">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
            <property name="spacing">
             <number>13</number>
            </property>
            <property name="margin">
             <number>13</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnStreamGo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>96</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Go</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>00</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string> hour </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>00</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string> min </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>00</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string> sec</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
         <zorder>label_3</zorder>
         <zorder></zorder>
         <zorder></zorder>
         <zorder>label_2</zorder>
         <zorder></zorder>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editRtmpUrl</tabstop>
  <tabstop>btnStreamGo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCamPlay</sender>
   <signal>clicked()</signal>
   <receiver>btnCamStop</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCamPlay</sender>
   <signal>clicked()</signal>
   <receiver>btnCamPlay</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCamStop</sender>
   <signal>clicked()</signal>
   <receiver>btnCamStop</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCamStop</sender>
   <signal>clicked()</signal>
   <receiver>btnCamPlay</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMinimize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCamPlay</sender>
   <signal>clicked()</signal>
   <receiver>btnCamStop</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCamStop</sender>
   <signal>clicked()</signal>
   <receiver>btnCamPlay</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamError</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioCamError</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamError</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCamError</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamIdle</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioCamIdle</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamIdle</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCamIdle</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamReady</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioCamReady</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamReady</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCamReady</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamAir</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioCamAir</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamAir</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCamAir</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioCamNone</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioCamNone</sender>
   <signal>toggled(bool)</signal>
   <receiver>labelCamNone</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
